app/assets/javascripts/set-ga-client-id-on-form.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+  'use strict'
+
+  window.GOVUK = window.GOVUK || {}
+  var GOVUK = window.GOVUK
+
+  function SetGaClientIdOnForm (options) {
+    if (!options.$form || !window.ga) { return }
+
+    var form = options.$form
+
+    window.ga(function(tracker) {
+      var clientId = tracker.get('clientId')
+      var action = form.attr('action')
+      form.attr('action', action + "?_ga=" + clientId)
```

Comment:
I guess this won't work as intended if the action URL already includes query parameters. Do you know if any action URLs include query parameters?

Reply:
The form currently submits without query parameters for all URLs (apart from the `_ga` param).

https://github.com/alphagov/government-frontend/pull/1483/files#diff-1e002ab60ce763d5443df324a62b2bcfR22

I checked the service sign in pages published by publisher from https://github.com/alphagov/publisher/tree/master/lib/service_sign_in and they seem to work (some of those rendered by the Heroku app are linked to in the description).

I considered using `url = new URL(action)` and then do something like `url.searchParams.set('_ga', 'clientId')` which would handle the case of multiple query params. But the constructor isn't supported by Internet Explorer (https://developer.mozilla.org/en-US/docs/Web/API/URL#Browser_compatibility).

Reply:
Ok :+1: